Automattic/wp-calypso
# Paid Stats: Ensure correct user currency for the pricing slider

Related to #

## Proposed Changes

There was an issue of changing currency when users saw the pricing slider:

https://github.com/Automattic/wp-calypso/assets/6869813/c661babf-c3c9-4991-bbd6-55c4fe204b65

* Get the currency code before the wizard is rendered.

## Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to `/stats/purchase/{site-slug}`.
* Select the `Personal site`.
* Ensure the currency of the pricing slider is always correct.

## Pre-merge Checklist
